de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/StoppableEditorFileListener.java

```diff
@@ -60,9 +59,9 @@ public void fileOpened(
         Editor editor = editorManager.getLocalEditorHandler()
             .openEditor(virtualFile, false);
 
-        SPath sPath = virtualFileConverter.convertToPath(virtualFile);
+        SPath sPath = VirtualFileConverter.convertToSPath(virtualFile);
 
-        if (sPath != null && editor != null) {
+        if (sPath != null && SessionUtils.isShared(sPath) && editor != null) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYNB10EcSutaxLwSy&open=AWcxYNB10EcSutaxLwSy)

[ISSUE_KEY]: AWcxYNB10EcSutaxLwSy